data/assets/nightsky/galactic_band.asset

```diff
@@ -60,22 +59,23 @@ local ToggleGalacticBand = {
 }
 
 asset.onInitialize(function()
-  openspace.addSceneGraphNode(GalacticBand)
+  openspace.addSceneGraphNode(GalacticLine)
   openspace.action.registerAction(ShowGalacticBand)
```

Comment:
Yes, it should 👍  I think I skipped it earlier to avoid creating too many breaking changes late in the release cycle of 0.20.0, but by now there are so many breaking changes anyway so it maybe does not matter.  

Thoughts, @alexanderbock ?  Since we want this in 0.20.1

Reply:
I also skipped renaming the assets, which all still have the "band" in the name while they are really lines.  Should we update these too?